finders/metadata/aaib-reports.json

```diff
@@ -6,6 +6,9 @@
   "name": "Air Accidents Investigation Branch reports",
   "beta": true,
   "beta_message": "Until early 2015, the <a href='http://www.aaib.gov.uk/home/index.cfm'>AAIB website</a> is the main source for AAIB reports",
+  "filter": {
+    "filter_document_type": "aaib_report",
```

Comment:
Given that [Finder Frontend prefixes filter params with `filter_`](https://github.com/alphagov/finder-frontend/blob/master/lib/finder_frontend.rb#L105), should this just be `document_type`?